_includes/breadcrumbs.html

```diff
@@ -1,12 +1,19 @@
 ﻿{% assign parent_dirs=page.url | replace_first: '/', '' | split: '/' %}
 {% assign link='/' %}
-<ol class="breadcrumb">
+<ol class="breadcrumb" vocab="http://schema.org" typeof="BreadcrumbList">
+    {% assign position = 1 %}
     {% for dir in parent_dirs %}
         {% if forloop.last %}
-            <li class="active">{{ dir }}</li>
```

Comment:
I forgot to check before I merged this, so I don't know if this was a result of the changes here or the major changes you made previously, but in breadcrumbs, the active page appears as an active link in the breadcrumbs.

It looks like you aren't doing anything with the `class` variable in liquid after you are assigning it. I'm guessing that is what broke it.